File: no-require-server/CoderByte Challenges/JavaScript/challenge_3.js
//reverseString
//an application to reverse a given string(apple-- > elppa);

function FirstReverse(str) {
    strArr = [];
    for (var counter = 0; counter < str.length; counter++) {
        strArr[counter] = str[counter];
    }
    strArr = strArr.reverse();
    str = "";
    for (var counterTwo = 0; counterTwo < strArr.length; counterTwo++) {
        str += strArr[counterTwo];
    }
    return str;
}
FirstReverse(readline()); 

File: no-require-server/CoderByte Challenges/Python/challenge_3.py
# reverseString
# an application to reverse a given string (apple --> elppa);

def FirstReverse(str):
    letter_list = [];
    for letter in str:
        letter_list.append(letter);
    letter_list = reversed(letter_list);
    str = "".join(letter_list);
    return str

while True:
    user_input = input("Type a string to get it back reversed: ")
    print(FirstReverse(user_input));

